_sass/pages/_what_is_login.scss

```diff
@@ -24,24 +26,51 @@
       margin-top: 2rem;
     }
   }
+
+  .text-column {
+    display: block;
+    margin-top: 2rem;
+    padding: 1rem 0;
+  }
+
+  .participating-agencies {
+    background: url(../img/what-is-login/how-it-works/how-it-works1.png) top left no-repeat;
```

Comment:
do we have 2x assets for this? or SVG?

Reply:
Looks like we could also shrink these a little using something like https://squoosh.app/ . I see 32% reduction on this image with the OxiPNG option.

Reply:
> do we have 2x assets for this? or SVG?

I can get those versions of the assets

Reply:
> > do we have 2x assets for this? or SVG?
> 
> I can get those versions of the assets

Now that we have those images, can we swap them out based on pixel density? I expect Zach's point is about how the original image could appear blurry on higher-quality displays.

We could [use a media query](https://css-tricks.com/snippets/css/retina-display-media-query/) to use the `@2x` instead on higher pixel density, or just using the SVG would scale nicely everywhere. I notice the SVG is a bit large (19.2kb), but not too bad. Simpler to use too, since it should just be as simple as replacing the file extension here:

```suggestion
    background: url(../img/what-is-login/how-it-works/how-it-works1.svg) top left no-repeat;
```

Whatever we decide, can we also remove the images that we're not using? We can add them back later if we need them.